### ISSUE TITLE
Add ability to change color for inactive windows

### ISSUE THREAD

**Issue by medhefgo** [mozilla/FirefoxColor#965]
There currently is no way to tell the active from inactive windows if the menu bar is disabled in firefox. Colors for "inactive window tab" and "inactive window tab text" would be nice to have.

**Comment by altano** [comment]
I'm new to theming but I stumbled into this issue and I think you want `frame_inactive` which Firefox Color has

**Comment by medhefgo** [comment]
I guess you're correct on frame inactive, I must've missed that. Though, you still cannot change the text color for inactive windows.

**Comment by altano** [comment]
> Though, you still cannot change the text color for inactive windows.

Correct, I don't believe that text is target-able/addressable. Same with text on active buttons, text for hovered tabs, etc.
